toolkit/tools/pkg/imagecustomizerlib/customizeselinux.go

```diff
@@ -104,6 +114,12 @@ func selinuxSetFiles(selinuxMode imagecustomizerapi.SELinuxMode, imageChroot *sa
 
 	logger.Log.Infof("Setting file SELinux labels")
 
+	_, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "set_selinux_files")
+	span.SetAttributes(
+		attribute.String("selinux_mode", string(selinuxMode)),
```

Comment:
We probably don’t need to log the same information multiple times.

Reply:
removed